### PR TITLE
Added image to geode-mods topic

### PR DESCRIPTION
**Pull request by TheBearodactyl**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by TheBearodactyl** [comment]
```
 1) Failure:
topics::egui topic#test_0011_has valid aliases [/home/runner/work/explore/explore/test/topics_test.rb:132]:
'egui_rs' must be between 1-35 characters, start with a letter or number, include only lowercase letters, and may include hyphens
```

the fuck

**Comment by ahpook** [comment]
Rebase onto the latest `main` @ e095255 and the tests should pass. Sorry 'bout that.

**Comment by TheBearodactyl** [comment]
> Rebase onto the latest `main` @ [e095255](https://github.com/github/explore/commit/e095255590c23d9d50b8eae25d6f73a339100daf) and the tests should pass. Sorry 'bout that.

@ahpook oki

**Comment by TheBearodactyl** [comment]
@ahpook Ok, it should be good now

File: topics/geode-mods/index.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ created_by: The Bearodactyl
 display_name: Geode Mods
 short_description: This topic is for people to advertise Geode mods.
```
Comment (ahpook) [inline topics/geode-mods/index.md:4]: Usually the short description has a little context about the topic - how about:
```suggestion
short_description: Geode is a modding framework for the rhythm platformer game Geometry Dash
```

**Review by ahpook — APPROVED** [review]
LGTM but feel free to address the comment I left. It's a suggestion to change the pre-existing part of the file so it wouldn't be fair to block this PR on it.

File: topics/geode-mods/index.md (hunk 1 above)
Comment (TheBearodactyl) [reply topics/geode-mods/index.md:4]: yeah, i could do that :D

**Pull request merged**